### ISSUE TITLE
Incompatible to Django 1.7

### ISSUE THREAD

**Issue by corecache** [jbittel/django-mama-cas#17]
Hey there,

Thanks a bunch for your work. Found the following incompatibility towards Django 1.7:

```
Traceback (most recent call last):
  File "/var/lib/sso/mama_cas/forms.py", line 41, in clean
    self.user = authenticate(username=username, password=password)
  File "/usr/lib/python2.7/dist-packages/django/contrib/auth/__init__.py", line 51, in authenticate
    for backend in get_backends():
  File "/usr/lib/python2.7/dist-packages/django/contrib/auth/__init__.py", line 26, in get_backends
    backends.append(load_backend(backend_path))
  File "/usr/lib/python2.7/dist-packages/django/contrib/auth/__init__.py", line 20, in load_backend
    return import_string(path)()
  File "/usr/lib/python2.7/dist-packages/django/utils/module_loading.py", line 26, in import_string
    module = import_module(module_path)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/lib/python2.7/dist-packages/django_auth_ldap/backend.py", line 59, in <module>
    from django.contrib.auth.models import User, Group, Permission, SiteProfileNotAvailable
ImportError: cannot import name SiteProfileNotAvailable
```

**Comment by corecache** [comment]
Nevermind, I see this is a bug of django_auth_ldap and none of yours.

**Comment by jbittel** [comment]
That's correct, the latest versions of MamaCAS should fully support Django 1.7. I'm guessing you're running an older version of django-auth-ldap, as this issue appears to have been resolved in 1.2.0. I'm currently running the latest version of django-auth-ldap with Django 1.7 with no problems.

Thanks!
